### ISSUE TITLE
Illegal string offset 'hex'

### ISSUE THREAD

**Issue by drakakisgeo** [thujohn/pdf-l4#21]
When I am trying to create a PDF I get this error. Some value in the css ( most probably a string value and not a HEX) creates this problem. Any solutions?

ErrorException

Illegal string offset 'hex'
open: /var/www/billit/vendor/thujohn/pdf/src/Thujohn/Pdf/dompdf/include/style.cls.php

```
if ( is_null($col) )
$col = self::$_defaults["color"];
//see __set and __get, on all assignments clear cache, not needed on direct set through __set
$this->_prop_cache["color"] = null;
$this->_props["color"] = $col["hex"];
}
```

**Comment by thujohn** [comment]
The only solution I found is adding a condition :

``` php
if (is_array($col))
     $this->_props["color"] = $col["hex"];
```

But I'm thinking to switch version.
This problem has been solved in 0.6 but it's still in beta. 0.5.2 is pretty old but stable...

**Comment by drakakisgeo** [comment]
Thanks that did the trick. Now I have unicode issues. Damn!
